tests/PageObjects/RemoteMic/RemoteMicSongLanguagesPage.ts

```diff
@@ -0,0 +1,56 @@
+import { Browser, BrowserContext, expect, Page } from '@playwright/test';
+
+export class RemoteMicSongLanguagesPagePO {
+  constructor(
+    private page: Page,
+    private context: BrowserContext,
+    private browser: Browser,
+  ) {}
+
+  public getLanguageButton(language: string) {
+    return this.page.getByTestId(language).locator('span');
+  }
+
+  public async selectLanguage(language: string) {
+    await this.getLanguageButton(language).click();
+  }
+
+  public async expectSongLanguageToBeSelected(language: string) {
+    await expect(this.getLanguageButton(language)).toHaveClass('text-active');
```

Comment:
it's better to define a `data-test` or `data-active` attribute instead. You won't need to locate elements by `span` etc.